pkg/multiTenancyPlugins/authorization/volumes.go

```diff
@@ -40,21 +38,21 @@ func filterVolumes(r *http.Request, rec *httptest.ResponseRecorder) []byte {
 	}
 	return buf.Bytes()
 }
-func hostFSMountCheck(tenantId string, volumeBindings[] string) (error) {
-	for _,b := range volumeBindings {
-		if index := strings.Index(b,":"); index > -1 {
-		  v := b[0:index]
-		  if strings.Contains(v,"/") {
-			return errors.New(HOST_FS_MOUNT_NOT_ALLOWED)
-		  }
+func hostFSMountCheck(tenantId string, volumeBindings []string) error {
+	for _, b := range volumeBindings {
+		if index := strings.Index(b, ":"); index > -1 {
+			v := b[0:index]
```

Comment:
regexp is very powerful, but also very very error prone.  Ezra: what do you prefer?